scripts-2.0/components/ContextMenu/Error/NoInternetError/index.tsx

```diff
@@ -0,0 +1,49 @@
+const browser = require('webextension-polyfill');
+import * as React from "react";
+import {useEffect, useState} from "react";
+import ContextMenuOverlay from "../../../ContextMenuOverlay";
+import Button from "../../../common/Button";
+import translate from "../../../../Translator";
+// @ts-ignore
+import imageSrc from "../../../../../images/no-internet.svg";
```

Comment:
tu i we wszystkich innych wielokropkowych ścieżkach nie możemy jakoś zaczynać od roota projektu ?